src/pyflexplot/data/presets/opr/icon-ch1-ctrl/all_shp.toml

```diff
@@ -28,22 +28,22 @@ time = -1
 combine_species = true
 
 [_base._concentration."_shp+"]
-outfile = "concentration_site-{release_site}_species-{species_id}_domain-{domain}_lang-{lang}_ts-{time_step}.shp"
+outfile = "{release_start}_{release_site}_FLEXPART-{model}_Dispersion_Konzentration_{time_step}.shp"
```

Comment:
maybe we should keep species_id in the name

Reply:
done